### PR TITLE
Markdown formatting fixes bonanza

### PR DESCRIPTION
**Pull request by marcsadler**
Addresses #152 

In addition:
- Remove separator line when it directly precedes h2
- Change remaining legacy bold text headings to h4 (apologies for not catching all of them last time round)
- Use the same bullet symbol for lists throughout
- Move instructions on how to install ectool from faq to "Using Ectool" page
- Change some tip container headings to "note" for general aesthetics
- Where it does not refer to terminal input/output, change monospace formatted text to normal text formatting in quotation marks
- Add nice Unicode symbols to ChromeOS VT-x keyboard shortcut references
- Simplify overly verbose instructions
- Spelling

### CODE REVIEW

File: src/docs/firmware/recovery-mode.md [hunk 1]
```diff
@@ -4,21 +4,21 @@ next: false
...
 ::: danger
 Do not continue if your device is currently owned or enrolled by a organization, such as a school or employer. Unless you were given permission by your organization, its a crime.
 ::: 
 
-Recovery mode is used to restore the OS and RW regions of the firmware. It gets triggered when an issue with the device is detected, or you manually start it.
-### Entering Recovery Mode
-<br>
+## Entering Recovery Mode
 
 #### Chromebooks
```
Comment (ninelore) [inline src/docs/firmware/recovery-mode.md:23]: h2 -> h4

File: src/docs/firmware/index.md [hunk 2]
```diff
@@ -13,28 +13,28 @@ Do not continue if your device is currently owned or enrolled by a organization,
...
-  * Your CPU model and generation
-  * Your storage devices (HDD/SSD, eMMC, NVMe, etc.)
-  * **Your boardname**
-    * For example, the boardname for Asus Chromebook C302CA is `Cave`
-  * If you don't know where to find this info, take a look at the [System Info Page](system-info.md).
+    * Your CPU model and generation
+    * Your storage devices (HDD/SSD, eMMC, NVMe, etc.)
+    * **Your boardname**
+        * For example, the boardname for Asus Chromebook C302CA is `Cave`
+    * If you don't know where to find this info, take a look at the [System Info Page](system-info.md).
```
Comment (ninelore) [inline src/docs/firmware/index.md:22]: ul indent should always be 2 spaces

**Review by ninelore — COMMENTED** [review]
https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

File: src/docs/firmware/write-protect.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ You do not have to disable write protect if flashing [RW_Legacy](about.md#rw_leg
...
 
 Depending on your device, you will need to do **one** of the following:
-- Take out a WP screw
-- [Unplug your battery](battery.md)
-- Bridge two jumpers
-- [Use a SuzyQable](suzyq.md) (A cable that enables CCD (Closed Case Debug))
+* Take out a WP screw
+* [Unplug your battery](battery.md)
+* Bridge two jumpers
+* [Use a SuzyQable](suzyq.md) (A cable that enables CCD (Closed Case Debugging))
```
Comment (ninelore) [inline src/docs/firmware/write-protect.md:17]: Both * and - are valid markdown ul. This has no effect.

File: src/docs/firmware/recovery-mode.md (hunk 1 above)
Comment (ninelore) [reply src/docs/firmware/recovery-mode.md:23]: multiple files with this issue

File: src/docs/reverting/making-recovery-usb.md [hunk 4]
```diff
@@ -19,34 +21,19 @@ There are two methods depending on the Operating System Installed.
...
 
-2. Open a terminal and run `lsblk`.
+Use [Rufus](../installing/bootableusb.md#flashing-using-rufus-windows) to flash the recovery .bin file to USB but make sure to flash in "dd" mode.
 
-3. Find your USB drive. USB drives usually appear as sdXX, while SD Cards usually appear as mmcblkX.
-    - If your using a SD card adapter, It might appear as sdXX instead.
+### macOS/ChromeOS/Linux
 
-4. Run `sudo dd if=PLACEHOLDER of=/dev/PLACEHOLDER bs=4M status=progress` but replace the placeholders with the path to the recovery image and USB drive respectivly.
+Use [dd](../installing/bootableusb.md#flashing-with-dd-linux-macos) to flash the recovery .bin file to USB.
```
Comment (ninelore) [inline src/docs/reverting/making-recovery-usb.md:39]: why did you remove a big chunk of info on this page?

File: src/docs/firmware/index.md (hunk 2 above)
Comment (ninelore) [reply src/docs/firmware/index.md:22]: multiple files with this issue

**Review by ninelore — CHANGES_REQUESTED** [review]

File: src/docs/firmware/write-protect.md (hunk 3 above)
Comment (marcsadler) [reply src/docs/firmware/write-protect.md:17]: Do you and the project have a preference?

Comment (ninelore) [reply src/docs/firmware/write-protect.md:17]: No preference. Tbh this is a FYI instead of a change request

**Comment by ninelore** [comment]
in 85be021 you changed alot of intents to 3sp. Was this intentional?

**Comment by marcsadler** [comment]
> No SEO-relevant issues left.
> 
> [Markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) still shows alot of trivial issues in regards to #87. Its up to you if you want toi fix em aswell.
> 
> Other than that: LGTM

I'll do that in a separate PR when I have some time, I don't really use VSCode but I'll check out the markdown linter.

**Comment by marcsadler** [comment]
Apologies, fixed some spelling I just noticed.

**Review by ninelore — APPROVED** [review]

**Review by WeirdTreeThing — APPROVED** [review]
